Sources/SwiftParser/Statements.swift

```diff
@@ -1241,6 +1280,8 @@ extension Parser.Lookahead {
       .switchKeyword?,
       .poundAssertKeyword?:
       return true
+    case .tryKeyword?:
+      return true
```

Comment:
seems like tightening this up will resolve some of the recovery knock-on effects elsewhere